database/README.md

```diff
@@ -32,3 +32,10 @@ to remove them or
 sbt flywayBaseline 
 ```
 to set the current state as the baseline.
+
```

Comment:
This exact information is already in the readme file. Remove it.

Reply:
Thanks @salamonpavel  will remove the file from the PR.